internal/unsecure/collections.go

```diff
@@ -0,0 +1,169 @@
+/*******************************************************************************
+ * Copyright 2019 VMWare Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *******************************************************************************/
+package unsecure
+
+import (
+	"github.com/globalsign/mgo"
+	"github.com/globalsign/mgo/bson"
+
+	"github.com/edgexfoundry/docker-edgex-mongo/internal/pkg"
+)
+
```

Comment:
It seems like all of this logic will be the same whether secured or not. The indexing and collection definitions don't depend on security. Consider putting this somewhere outside of the `unsecure` package for easy re-use.